docs/modules/ROOT/pages/tmail-backend/configure/index.adoc

```diff
@@ -311,3 +311,63 @@ Centralize logging third party like FluentBit can tail from logging’s file the
 
 
 Those logs can then dinamically be parsed by the link:https://grafana.com/oss/loki/[Loki] stack.
+
+== Ecosystem Discovery
```

Comment:
IMO, this extension is not a `JMAP extensions for TMail` 
then I think put document here is better than `jmap-extensions` package